CharlieBackend.Api/Controllers/CoursesController.cs

```diff
@@ -42,14 +42,14 @@ public async Task<ActionResult<CreateCourseDto>> PostCourse(CreateCourseDto cour
         }
 
         /// <summary>
-        /// Gets all cources
+        /// Gets cources
         /// </summary>
         /// <response code="200">Successful return of list of courses</response>
         [Authorize(Roles = "Admin, Mentor, Secretary, Student")]
-        [HttpGet]
-        public async Task<IList<CourseDto>> GetAllCourses()
+        [HttpGet("isAvtive")]
+        public async Task<IList<CourseDto>> GetCourses(bool? isActive)
```

Comment:
* `activeOnly` should be a more suitable and descriptive name for this argument
* please add parameter description:
```cs
/// <param name="activeOnly">description</param>
```